nvme-print-stdout.c

```diff
@@ -1592,44 +1646,33 @@ static void stdout_single_property(int offset, uint64_t value64)
 
 	if (!human) {
 		if (nvme_is_64bit_reg(offset))
-			printf("property: 0x%02x (%s), value: %"PRIx64"\n",
+			printf("property: 0x%04x (%s), value: %"PRIx64"\n",
 			       offset, nvme_register_to_string(offset), value64);
 		else
-			printf("property: 0x%02x (%s), value: %x\n", offset,
+			printf("property: 0x%04x (%s), value: %x\n", offset,
 			       nvme_register_to_string(offset), value32);
 		return;
 	}
 
-	switch (offset) {
-	case NVME_REG_CAP:
-		printf("cap : %"PRIx64"\n", value64);
-		stdout_registers_cap((struct nvme_bar_cap *)&value64);
-		break;
-	case NVME_REG_VS:
-		printf("version : %x\n", value32);
-		stdout_registers_version(value32);
-		break;
-	case NVME_REG_CC:
-		printf("cc : %x\n", value32);
-		stdout_registers_cc(value32);
-		break;
-	case NVME_REG_CSTS:
-		printf("csts : %x\n", value32);
-		stdout_registers_csts(value32);
-		break;
-	case NVME_REG_NSSR:
-		printf("nssr : %x\n", value32);
-		printf("\tNVM Subsystem Reset Control (NSSRC): %u\n\n", value32);
-		break;
-	case NVME_REG_CRTO:
-		printf("crto : %x\n", value32);
-		stdout_registers_crto(value32);
-		break;
-	default:
-		printf("unknown property: 0x%02x (%s), value: %"PRIx64"\n",
-		       offset, nvme_register_to_string(offset), value64);
-		break;
+	if (nvme_is_fabrics_reg(offset)) {
```

Comment:
Sorry not necessary the case so deleted since probably tried to use the function by the get-reg command at first but not used actually for now I think.